Octokit/Clients/PullRequestsClient.cs

```diff
@@ -43,8 +49,8 @@ public PullRequestsClient(IApiConnection apiConnection) : base(apiConnection)
         /// </remarks>
         public Task<PullRequest> Get(string owner, string name, int number)
         {
-            Ensure.ArgumentNotNullOrEmptyString(owner, "owner");
-            Ensure.ArgumentNotNullOrEmptyString(name, "name");
+            Ensure.ArgumentNotNullOrEmptyString(owner, nameof(owner));
```

Comment:
Just realized I did this in the wrong file. Will do it in right file and leave this here for now since you said you're standardizing on it anyway